.github/workflows/nightly.yml

```
@@ -0,0 +1,36 @@
+name: Nightly DDC check
+
+on:
+  schedule:
+    # Trigger at 4:30 am UTC every day
+    - cron: '30 4 * * *'
+  push:
+    branches: [ebourne_nightly]
+
+jobs:
+  cpu_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toolchain: ['tests_toolchain', 'tests_release_toolchain', 'tests_release_omp_toolchain']
+    container:
+      image: ghcr.io/gyselax/gyselalibxx_env:latest
+      options: --user root
+    name: CPU Test
+    steps:
+      - name: Checkout gyselalibxx
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Checkout recent DDC version
+        run: |
+          cd vendor/ddc
+          git fetch
+          git checkout main
```

```suggestion
        uses: actions/checkout@v4
        with:
          repository: https://github.com/CExA-project/ddc.git
          path: vendor/ddc
```

I think it would be better. I did what you have in DDC, if you look at "Checkout Kokkos devel branch" in https://github.com/CExA-project/ddc/actions/runs/14394358248/job/40369240967, I cannot get the exact commit being tested. I find it unfortunate when reporting issues.

---

I don't know, apparently I removed the `vendor/ddc` for https://github.com/CExA-project/ddc/blob/main/.github/workflows/gyselalibxx.yml before cloning. I don't think I checked I actually needed to remove it.

---

seems to be working https://github.com/CExA-project/ddc/actions/runs/14406301085/job/40403582521